keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -426,11 +426,18 @@ This means that an existing pod which is not valid according to the current
 
 #### Other Pod Subresources
```

Comment:
updated this to explicitly select which subresources are excluded (as requested by @deads2k during the implementation code review) to make this fail safe if new subresources that allow writing containers or relevant fields are introduced